docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to create a project that you can build and run remotely. 
+
+If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow [Import an existing local project to the cloud](##import-an-existing-local-project-to-the-cloud).
```

Comment:
Link to section within the markdown does not work.

Reply:
Add **Note** at the start of this paragraph to ensure user does not confuse these instructions with Creating a New Project, introduced in the previous paragraph. (This will become a callout box once that function has been added).

Reply:
Any ideas as to why the link doesn't work? It works for me in VS Code, but only there. I don't know why it works in the editor, but not on the link that I render after inputting the `./serve.sh` command...